### ISSUE TITLE
[admin-settings] keyboard navigation breaks after deleting an item

### ISSUE THREAD

**Issue by JammingBen** [owncloud/web#10186]
## Steps to reproduce
1. Go to the users list in the admin settings
2. Select a user and delete it
3. Try to navigate via keyboard navigation (e.g. arrow keys up and down)

## Expected behavior
Keyboard navigation works just like before deleting the user.

## Actual behavior
Keyboard navigation does not work. The same happens for groups and spaces.
